_preload.lua

```diff
@@ -125,6 +125,27 @@ premake.api.register {
 	kind = "string-list"
 }
 
+--
+-- Specify a list of custom options to send to the Qt lrelease command line.
+--
+premake.api.register {
+	name = "qttsargs",
+	scope = "config",
+	kind = "string-list"
+}
+
+--
+-- Specify the path, relative to the current script, where the qm files generated
+-- by Qt will be created. If this command is not used, the default behavior
+-- is to generate those files near the ts files.
+--
```

Comment:
The comment will need to reflect the default changes I mentioned in a previous comment :)

Reply:
Changed.